### ISSUE TITLE
[documentation] Fix typos

### ISSUE THREAD

**Issue by cassgvp** [cassgvp/WIN-Open-Neuroimaging-Community#7]
There are many many typos in these pages 🤦‍♀️. 

Anyone who spots one is invited to submit a correction. This could be done as an exercise in the hack day on how to contribute. 

- [ ] on the main page it say open WIN tools but the 'open WIN' is dropped for the community, ambassadors and events . On the left-hand bar all the links are with 'open WIN'
- [ ] in the open WIN community tab - Find out about the Open WIN Community, what *we're* doing, and how you can be a part of this effort.
- [ ] in the open WIN community tab - We *are* committed

**Comment by jvohryzek** [comment]
I agree:) very nice way of making your first corrections/issues on github.

Let me kick off with a few typos I have been collecting in my basket as I was walking through the documentation forest.

So far just as comments on this issue
- [ ]  on the main page it say open WIN tools but the 'open WIN' is dropped for the community, ambassadors and events . On the left-hand bar all the links are with 'open WIN'
- [x] in the open WIN community tab - Find out about the Open WIN Community, what **we're** doing, and how you can be a part of this effort.
- [x]  in the open WIN community tab - We **are** committed

**Comment by cassgvp** [comment]
Thank you for these! I'm going to add them as actions on the original issue and continue to add any further which are identified. 

🌲🌳🧺🌴🎄 😁

**Comment by JasleenKJolly** [comment]
- [x] On the roadmap it keeps saying work in progresses and should be progress

**Comment by jescaf** [comment]
Hi I am going to start fixing these typos.

**Comment by cassgvp** [comment]
Thanks @jescaf ! Merged the first set of typos in 2efc5f56b3177d13acaf0ac1889e1379b92a6c88 :)

**Comment by cassgvp** [comment]
Awesome @jescaf ! More typos fixed on d2b00c6678d604a22843bddaa8714b2a2fa28471
